app/webpacker/styles/sections/hero.scss

```diff
@@ -400,5 +400,5 @@ $mobile-cutoff: 800px;
 	&.green-yellow { @include colours($green, $yellow); }
 	&.yellow-green { @include colours($yellow, $green); }
 	&.yellow-yellow { @include colours($yellow, $yellow); }
-
+  &.grey-pink { @include colours($grey, $pink); }
```

Comment:
@spencerldixon do you know why this change is still showing as a change as part of this PR?

Reply:
This is already in main. I suspect it's hanging around here because this branch was created before we merged this line in. To be on the safe side you could merge main into this branch to bring everything in line, and this should disappear from the list of changed files